core/lib/data/network/download/download_manager.dart

```diff
@@ -79,31 +82,40 @@ class DownloadManager {
     return streamController.stream.first;
   }
 
-  Future<bool> downloadFileForWeb(String downloadUrl, String filename, String basicAuth) async {
-    final headerParam = Map<String, String>();
-    headerParam[HttpHeaders.authorizationHeader] = basicAuth;
-    headerParam[HttpHeaders.acceptHeader] = DioClient.jmapHeader;
+  Future<bool> downloadFileForWeb(
```

Comment:
how about make the download progress display in browser?

Reply:
1. To download progress display in browser (Specifically Chrome) we need `downloadUrl public` (download url does not use `authentication`)
2. Currently download attachment requires `accessToken` or `basicAuth`.
=> I don't know if the server side currently supports getting `downloadUrl public` so we can do that?

Reply:
create another ticket #654 for it

Reply:
The dowload response should have a content length header allowing you to know the size of what you need to download.

Then you just need to count the downloaded bytes.

> => I don't know if the server side currently supports getting downloadUrl public so we can do that?

Thats off spec...